options.js

```diff
@@ -123,9 +176,93 @@ function getVals() {
 	chrome.extension.getBackgroundPage().rcxMain.config.copySeparator = localStorage["copySeparator"];
 	chrome.extension.getBackgroundPage().rcxMain.config.maxClipCopyEntries = localStorage["maxClipCopyEntries"];
 	chrome.extension.getBackgroundPage().rcxMain.config.popupLocation = localStorage["popupLocation"];
+    
+}
+
+
+
+// RTK
+  /**
+ * CSVToArray parses any String of Data including '\r' '\n' characters,
+ * and returns an array with the rows of data.
+ * @param {String} CSV_string - the CSV string you need to parse
+ * @param {String} delimiter - the delimeter used to separate fields of data
+ * @returns {Array} rows - rows of CSV where first row are column headers
+ */
+function CSVToArray (CSV_string, delimiter) {

```

Comment:
Is this referenced from somewhere, I see some similar functions on stackoverflow.